torch/testing/_internal/common_methods_invocations.py

```diff
@@ -85,13 +85,16 @@ def __init__(self, input, *, args=tuple(), kwargs=None, output_process_fn_grad=N
         self.kwargs = kwargs if kwargs is not None else {}
         self.output_process_fn_grad = output_process_fn_grad
 
+    def _is_tensorlist(self, arg):
+        return isinstance(arg, (tuple, list)) and len(arg) > 0 and isinstance(arg[0], torch.Tensor)
```

Comment:
This is probably fine. Just fyi I recently wrote a similar function:

https://github.com/pytorch/pytorch/blob/2d36b30a8c35cc5640e1c735583c105b7037f9ca/test/test_ops.py#L466

Reply:
I moved this method to testing._internal.common_utils to be reused in common_methods_invocation and test_ops